### DISCUSSION TITLE
New Android Photo & Videos Permission Policy

### DISCUSSION THREAD

**Discussion by ezescigo** [ivpusic/react-native-image-crop-picker#2080] [Q&A]
Hi all!

Regarding the new policies that Android will be applying next month: https://support.google.com/googleplay/android-developer/answer/14115180?hl=en, is this library affected some way? Does this library uses the Android Picker already?

Thank you!

**Comment by 1Jesper1** [comment]
See https://github.com/ivpusic/react-native-image-crop-picker/releases/tag/v0.42.0 implemented.
